src/jit/codegencommon.cpp

```diff
@@ -9406,9 +9422,6 @@ void                CodeGen::genFnEpilog(BasicBlock* block)
             compiler->unwindBegEpilog();
             unwindStarted = true;
         }
-        // mov FP into SP
-        inst_RV_RV(INS_mov, REG_SPBASE, REG_FPBASE);
-        compiler->unwindSetFrameReg(REG_FPBASE, 0);
```

Comment:
This is exactly the case. The 0 is the offset, if you use an add/sub to establish the fp.

---
In reply to: [57773432](https://github.com/dotnet/coreclr/pull/3966#discussion_r57773432) [](ancestors = 57773432)